packages/tools/pie-icons-webc/helpers/index.js

```diff
@@ -0,0 +1,5 @@
+const removeHyphenBeforeDigits = (name) => name.replace(/\-(\d+)/, '$1'); // eslint-disable-line no-useless-escape

```

Comment:
/**
 * Removes any hyphens from before digits in a string
 * This is used used example where we generate icon names from keys, such as over-18-filled-large
 * and without this the PascalCase generation returns Over-16Filled
 * @param {string} name - a string, for example an iconKey
 * @returns {string}
 */

Reply:
Dow we have any icons with the digits in the name?

Reply:
Interesting why it is not an issue for React and Vue packages that also use `pascalCase`

Reply:
I checked that in React library 
```
const handleComponentName = (name) => name.replace(/\-(\d+)/, '$1');
pascalCase(handleComponentName(iconKey))
``` 
 does the trick.

Maybe worth updating there as well for consistency

Reply:
oh nice! Vue also uses the helper function so I'll update both so they're consistent with react 👍

Reply:
Would it not be worth changing the React one to use the helper? That way we don't have the same regex repeated 3 times across the libraries?

@dandel10n maybe let's chat about Monday? I'm happy to help make this change in the PR so we can help get this through.